src/main/webapp/js/item_page_loader.js

```diff
@@ -0,0 +1,41 @@
+let map;

```

Comment:
Just FYI, this is in the global namespace, which means any code trying to use a "map" variable (including in other files) will either use or override yours.

This isn't a good practice for a variety of reasons, but I'm okay with it for now since this is more of a prototype and fixing it requires a bit of work they haven't prepared you for well.

Is a global variable required for the map to work properly?

Reply:
It's not required. Does putting it in function make it not global?